Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/directives/localizeOptions.Directive.js

```diff
@@ -0,0 +1,12 @@
+﻿angular.module("umbraco.directives").directive('localizeOption', function (localizationService) {
+    return {
+        restrict: 'A',
+        link: function ($scope, element, attrs, controller) {
+            var key = attrs.localizeOption;
+
+            localizationService.localize(key).then(function (value) {
+                element.text(value);
+            });
+        }
+    }
+});
```

Comment:
@warrenbuckley okay, if you search in https://github.com/umbraco/Umbraco-CMS/ for localizationService there are some directives.
umbcontrolgroup.directive.js:
https://github.com/umbraco/Umbraco-CMS/blob/5b9a98ad6ae9e63322c26f7b162204e34f7fcb54/src/Umbraco.Web.UI.Client/src/common/directives/html/umbcontrolgroup.directive.js

localize.directive.js:
https://github.com/umbraco/Umbraco-CMS/blob/5b9a98ad6ae9e63322c26f7b162204e34f7fcb54/src/Umbraco.Web.UI.Client/src/common/directives/util/localize.directive.js

So on this line https://github.com/bjarnef/Analytics/blob/Localize-labels/Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/partials/settings.html#L31 it can maybe just use ```localize"analytics_selectAnAccount"``` if the directive is available to the controller.